Modules/_io/bufferedio.c

```diff
@@ -452,8 +452,9 @@ _io__Buffered___sizeof___impl(buffered *self)
 }
 
 static int
-buffered_traverse(buffered *self, visitproc visit, void *arg)
+buffered_traverse(PyObject *op, visitproc visit, void *arg)
 {
+    buffered *self = (buffered*)op;
```

Comment:
Don't we have a macro for this cast? I'm almost sure I had seen one but maybe I actually dreamt of one.